src/localize.test.js

```diff
@@ -2,13 +2,13 @@ import localize from './localize';
 
 describe('localize', () => {
   test('accented', () => {
-    expect(localize('abcd')).toBe('ȧȧƀƈḓ');
+    expect(localize('abcd')).toBe('ȧȧȧƀƀƀƈḓ');
```

Comment:
With the new logic, shouldn't strings that are less than 10 characters be expanded by 250%? This is only expanded by about 50%.

Reply:
These should be 10 characters, you are definitely right, let me fix this.

Reply:
I forgot to push this check here 44adbba, it should be `ȧȧȧƀƀƀƈƈƈḓ` where 4 characters are translated to 10 chars, hence the total strings becomes 250%.